react-native/trainingTodoList/src/app/screens/Books/components/BookList/BookItem/index.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { View, Text, Image } from 'react-native';
+
+import CustomButton from '../../../../../components/CustomButton';
+
+import styles from './styles';
+
+const BookItem = props => {
+  const {
+    book: { author, title, image_url }
+  } = props;
+
+  const imageSource = image_url ? { uri: image_url } : require('./assets/not_available.jpg');
```

Comment:
U are import a asset in each render. Thats a big problem for the performance. import the just once and use the reference. Also, this `imageSouce` should be assigned the value only once, so convert this component to a class component, and define a instance of this class, something like `imageSource = this.props.image_url ? { uri: image_url } : notAvailableImg`